api/services/user_contact_service.py

```diff
@@ -148,3 +159,7 @@ def custom_fields_contidions(self, field_name, field_value):
     def mark_user_groups_as_inactive(self):
         models.UserGroup.query.mark_user_groups_as_inactive(self.user_phone)
         db.session.commit()
+
+    def mark_user_custom_fields_as_inactive(self):
+        models.UserCustomFields.query.mark_user_groups_as_inactive(self.user_phone)
```

Comment:
The function name and the action seem confusing to me. Could you explain the reason?

Reply:
corrected it